cypress-tests/cypress/e2e/PaymentMethodListUtils/Commons.js

```diff
@@ -114,7 +114,8 @@ export const bank_redirect_ideal_and_credit_enabled = [
 
 export const create_payment_body_with_currency_country = (
   currency,
-  billingCountry
+  billingCountry,
+  shippingCountry
```

Comment:
remove `shippingCountry` and rename `billingCountry` to something like, let's say `country`(?) and pass it as a parameter to both `billing` and `shipping` below.

Reply:
we need shipping and billing separately